src/components/DeviceInteraction/DeviceInteractionStep.js

```diff
@@ -0,0 +1,225 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import Box from 'components/base/Box'
+import { delay } from 'helpers/promise'
+
+import {
+  DeviceInteractionStepContainer,
+  SpinnerContainer,
+  IconContainer,
+  SuccessContainer,
+  ErrorDescContainer,
+  ErrorContainer,
+} from './components'
+
+export type Step = {
+  id: string,
+  title: string | (any => string),
+  desc: any,
+  icon: React$Element<*> | null,
+  run?: any => Promise<any> | { promise: Promise<any>, unsubscribe: void => any },
+  render?: (
+    { onSuccess: any => any, onFail: any => any, onRetry: any => any },
+    any,
+  ) => React$Element<*>,
```

Comment:
use `React$Node` instead of all your `React$Element<*>` . it encapsulate more cases (meaning for icon you don't even need | null )
see https://github.com/facebook/flow/blob/master/lib/react.js#L12